packages/devtools_shared/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 * Add the ability to send debug logs in DevTools server request responses. 
 * Add an optional positional parameter `logs` to the `ServerApi.serverError` method.
 * Include debug logs with the `ExtensionsApi.apiServeAvailableExtensions` API response.
+* Add parameter to `ServerApi.handle` to accept an instance of `Analytics` from 
+  `package:unified_analytics`.
+
```

Comment:
can you also add an entry for the new server apis that were added?

Reply:
yep, pushed just now